src/app/resources/page.tsx

```diff
@@ -1,26 +1,65 @@
-import React from "react";
+"use client";
+import { useState, useRef, useEffect } from "react";
+import gsap from "gsap";
+import Lenis from "lenis";
+
+import Resources from "@/components/Resources/Resources";
+import { PageHeader } from "@/components";
 
 const Page = () => {
+  const [selectedArticle, setSelectedArticle] = useState("all posts");
+
+  const articles = useRef<(HTMLElement | null)[]>([]);
+
+  const onClick = (article: string, index: number) => {
+    setSelectedArticle(article);
+    gsap.to(articles.current[index], {
+      scale: 1.1,
+      duration: 0.2,
+      ease: "power4.inOut",
+      onComplete: () => {
+        gsap.to(articles.current[index], {
+          scale: 1,
+          duration: 0.2,
+          ease: "power4.inOut",
+        });
+      },
+    });
+  };
+
+  useEffect(() => {
+    const lenis = new Lenis();
+
+    function raf(time: number) {
+      lenis.raf(time);
+      requestAnimationFrame(raf);
+    }
+
+    requestAnimationFrame(raf);
+  }, []);
+
   return (
     <>
-      <div>resouces</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>resources</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
-      <div>page</div>
+      <PageHeader header="resources" />
+      <section className="centered flex-col">
+        <nav>
```

Comment:
Do we need this nav while there are only 3 articles? Don't think Holly is planning to write anymore for the time being so maybe we can just leave this commented out. Also don't think we need the section wrapper, I think just nav is fine.

Reply:
Yeah I'm not sure why I used a nav link in the first place. I think in my head I thought the contents act as a navigation menu to another page. Article tag would be better.

I think I'm going to come back to this branch another time and render this server side. We can have a chat about this at some point but I think creating the articles as markdown files is the best way forward.